### PR TITLE
Add support for Django 2.0 to 3.0

### PR DESCRIPTION
**Pull request by kmichel-sereema**
This fixes a runtime issue with template names in subsegments (#168) and two failed imports during tests.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by kmichel-sereema** [comment]
Hi,

I fixed the issues I had when deploying xray with Django 3.0 but removed the commit in this PR enabling coverage with tox: I'm too confused about tox to create a correct configuration and only noticed #200 after pushing this code.

**Comment by srprash** [comment]
Hi @kmichel-sereema 
Thanks for the PR. Your tests for Python 3.6+ are failing because of a change in aiobotocore 0.12.0 a couple of days back where the `loop` parameter was dropped from the `get_session` api. I have pushed the changes to our current aiobotocore tests in [this PR](https://github.com/aws/aws-xray-sdk-python/pull/205). Your tests should be passing once this PR is merged.

File: aws_xray_sdk/ext/django/templates.py [hunk 1]
```diff
@@ -22,6 +23,10 @@ def xray_render(self, context):
         template_name = self.name or getattr(context, 'template_name', None)
         if template_name:
             name = str(template_name)
+            # SafeString are not properly serialized by jsonpickle,
+            # turn them back to str by adding a non-safe str.
+            if isinstance(name, SafeString):
```
Comment (srprash) [inline aws_xray_sdk/ext/django/templates.py]: Can you add a test case for this code path to maintain coverage?

Comment (kmichel-sereema) [reply aws_xray_sdk/ext/django/templates.py]: I added the test and updated the django version (the test wouldn't cover the case without updating django).

File: tox.ini [hunk 2]
```diff
@@ -19,7 +21,9 @@ deps =
     Flask-SQLAlchemy
     future
     # the sdk doesn't support earlier version of django
-    django >= 1.10, <2.0
+    py{27,34}-!django2-!django3,django1: django >= 1.10, <2.0
```
Comment (srprash) [inline tox.ini]: You won't probably need to specify the supported django version for each runtime. If you just do
`py{27,34,35,36,37,38}: django >= 1.10`, tox should automatically pick up the highest version for each python runtime. Please correct me if I'm wrong.

Comment (kmichel-sereema) [reply tox.ini]: You're right, tox is smart enough to rely on pypi classifiers.

Thanks for the suggestion.

**Pull request merged**